app/plugins/Files/js/loaders.js

```diff
@@ -87,6 +112,14 @@ addResultListener('ascii-loaded', function(result) {
 	update();
 });
 
+//Confirm deletion popup
+function confirmDelete(nickname) {
+	eID('confirm-delete').querySelector('.title').innerHTML = 'Confirm to delete';
```

Comment:
any reason why this isn't just defined in the HTML?
I would change the title to `'Confirm'`, and inside the popup `'Really delete ' + nickname + '?'`

also, in the future, try not to add multiple features in one pull request.